content/en/docs/reference/glossary/workload.md

```diff
@@ -1,24 +1,23 @@
 ---
-title: Workloads
+title: Workload
 id: workloads
 date: 2019-02-13
 full_link: /docs/concepts/workloads/
 short_description: >
-   Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 aka: 
 tags:
 - fundamental
-- core-object
 - workload
 ---
-  Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
```

Comment:
hello @jaypipes. A few thoughts:
- At first glance, it seems odd to have the `Workload` entry in the `Workload` tag selector of the glossary. The Workload concept includes the listed resources in that category.
- The  `short` description is succinct, though, is it your intention to make the definition generic and not k8s specific? The rest of the description expands to include the k8s resource objects.

Reply:
> hello @jaypipes. A few thoughts:
> 
>     * At first glance, it seems odd to have the `Workload` entry in the `Workload` tag selector of the glossary. The Workload concept includes the listed resources in that category.

Agreed. I've removed the workload tag from the workload entry.

>     * The  `short` description is succinct, though, is it your intention to make the definition generic and not k8s specific? The rest of the description expands to include the k8s resource objects.

Well, my intention was to keep the concept specific to Kubernetes by using the wording "running on Kubernetes" but to make the definition of "workload" reflect that it is a concept, not a specific /object/.